### PR TITLE
RestAssured

### PR DESCRIPTION
**Pull request by alszla**
Gist creation example

### CODE REVIEW

File: src/test/resources/CreateGist.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Create a gist
+  Scenario: User creates a gist on GitHub
+    Given a user is registered at GitHub
+      And user has content
+    When a user makes a post request
+    Then the create status code is 201
+      And response includes the following
+      | owner.login           | alszla            |
+      | owner.type            | User              |
```
Comment (titor-oopart) [inline src/test/resources/CreateGist.feature]: add empty line

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package com.fundation.cucumberRestAssured;
```
Comment (titor-oopart) [inline src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java:1]: add Copyright

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ */
+public class CreateGistStepdefs {
+    private Response response;
+    private ValidatableResponse json;
+    private RequestSpecification request;
+    private AuthenticationSpecification authentication;
+
+    private String GIST_ENDPOINT = "https://api.github.com/gists";
+
+
```
Comment (titor-oopart) [inline src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java]: delete this unnecessary line

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+/**
+ * @author Alejandro Sánchez Luizaga
+ * @version 1.0
+ */
+public class CreateGistStepdefs {
+    private Response response;
+    private ValidatableResponse json;
+    private RequestSpecification request;
+    private AuthenticationSpecification authentication;
+
```
Comment (titor-oopart) [inline src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java]: delete this unnecessary line

File: src/test/resources/CreateGist.feature (hunk 1 above)
Comment (alszla) [reply src/test/resources/CreateGist.feature]: Fixed

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java (hunk 2 above)
Comment (alszla) [reply src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java:1]: Hmmm, I guess we're not copyrighting our code yet, are we?

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java (hunk 3 above)
Comment (alszla) [reply src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java]: Fixed.

File: src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java (hunk 4 above)
Comment (alszla) [reply src/test/java/com/fundation/cucumberRestAssured/CreateGistStepdefs.java]: Fixed. Thanks for your review.

**Pull request merged**